crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -8,6 +8,7 @@ use base64::{
     prelude::BASE64_STANDARD,
 };
 use bytes::Bytes;
+use hex::FromHex;
```

Comment:
nit: move this import into the `try_from_raw_ref`, and also `use hex::FromHex as _;`